### ISSUE TITLE
Sample test depends on <h1> tag

### ISSUE THREAD

**Issue by ramijarrar** [xolvio/meteor-cucumber#155]
The sample feature that comes with this package depends on the presence of an h1 tag in the user's app on this line of code (sample_steps.js)

```waitForVisible('h1'). // WebdriverIO chain-able promise magic```

In cases where this tag is never used, the user (who is likely using this package for the first time) encounters an intimidating ```RuntimeError  ...``` in the html-reporter.

Also, in the cucumber log file:

``` RuntimeError: RuntimeError
     (ScriptTimeout:28) A script did not complete before its timeout expired.
     Problem: Timed out waiting for asyncrhonous script result after 504 m```

**Comment by samhatoum** [comment]
good call, thanks for reporting

**Comment by ramijarrar** [comment]
Fixed in #79
